helm/opendistro-es/Chart.yaml

```diff
@@ -26,4 +26,4 @@ name: opendistro-es
 sources:
 - https://pages.git.viasat.com/ATG/charts
 # Chart version
-version: 1.10.1
+version: 1.10.2
```

Comment:
Hi @madeleine666 this change is not related to add startupprobe.
Please consider making a new PR for such change.
Thanks.

Reply:
Hi @peterzhuamazon, 
my mistake, removed it.